client/src/game/shapes/utils.ts

```diff
@@ -156,6 +156,7 @@ export function deleteShapes(): void {
         l.removeShape(sel, SyncMode.FULL_SYNC);
         EventBus.$emit("SelectionInfo.Shape.Set", null);
         EventBus.$emit("Initiative.Remove", sel.uuid);
+        gameStore.removeMarker({ marker: sel.uuid, sync: true }); //delete marker.... even if it doesn't exist.
```

Comment:
Moved... 
Should there be a check to see whether the shape is marked or can I just try to remove the marker, even it might not exist? If I need to check first, I need to access (or create) the marker array which layer doesn't have direct access to, which means more references and arrays.